src/java/fp/v11/EditIssue.java

```diff
@@ -1,5 +1,3 @@
-package fp.v11;
-
```

Comment:
- First, `EditIssue` should be refactored away into `FootPrints11.editIssue()`.
- Seconf, the package `fp.v11` here was nearly correct. It should remain, and a subpackage `splints` should be created.